ui/DesignSystem/Component/Wallet/Receive.svelte

```diff
@@ -24,5 +28,7 @@
     </Text>
   {/if}
   <Title style="padding: 1.5rem 0 0.5rem 0;">Address</Title>
-  <Urn urn={accountId} />
+  <div class="center">
```

Comment:
We have `<Flex>` for that.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/731/commits/26a4ef75ad98cfb544650ab690ce9aa8ee106a80